op-conductor/conductor/service.go

```diff
@@ -766,16 +820,29 @@ func (oc *OpConductor) stopSequencer() error {
 	//
 	// To that end we allow to cancel the StopSequencer call if we're shutting down.
 	latestHead, err := oc.ctrl.StopSequencer(oc.shutdownCtx)
-	if err == nil {
-		// None of the consensus state should have changed here so don't log it again.
-		oc.log.Info("stopped sequencer", "latestHead", latestHead)
-	} else {
+	if err != nil {
 		if strings.Contains(err.Error(), driver.ErrSequencerAlreadyStopped.Error()) {
 			oc.log.Warn("sequencer already stopped", "err", err)
 		} else {
+			// If we can't stop the sequencer, maintain consistency by not touching Rollup boost
 			return errors.Wrap(err, "failed to stop sequencer")
 		}
+	} else {
+		oc.log.Info("stopped sequencer", "latestHead", latestHead)
+	}
+
+	// If we get here, sequencer is successfully stopped, now disable Rollup boost
+	if oc.rollupBoost != nil {
+		if err := oc.rollupBoost.SetExecutionMode(oc.shutdownCtx, client.ExecutionModeDisabled); err != nil {
+			oc.log.Error("failed to disable rollup boost execution mode", "err", err)
+			// this is a critical error, we should return an error and trigger an alert
+			// to preserve strong consistency, we should not continue to sequence
+			// this is treated same as failed to stop sequencer
+			return fmt.Errorf("failed to disable rollup boost execution mode: %w", err)
+		}
+		oc.log.Info("rollup boost disabled")
 	}
+
 	oc.metrics.RecordStopSequencer(err == nil)
 	oc.seqActive.Store(false)
```

Comment:
this needs to happen right after StopSequencer call, otherwise rollupboost failure will cause this code to never be called